scripts/cci-process-data.ts

```diff
@@ -0,0 +1,32 @@
+import { join } from "path";
+import { getOldJobNameWithoutSuffixes, getTestFiles, getTimingsFromJobsData, REPO_ROOT, saveTestTimings } from "./cci-utils";
+
+function main(): void {
```

Comment:
todo: add js docs

This function takes all tests suites & checks if we have timing data for them.
If we have timing data, we can use that to organize tests by duration.

This script saves the timing data to a file that is checked in; so we don't need to setup CCI tokens for every dev, someone with a CCI token should just be able to run the data scripts every few months.